src/Umbraco.Core/Deploy/ArtifactSignature.cs

```diff
@@ -1,17 +1,27 @@
 namespace Umbraco.Cms.Core.Deploy;
 
+/// <inheritdoc />
 public sealed class ArtifactSignature : IArtifactSignature
 {
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ArtifactSignature" /> class.
+    /// </summary>
+    /// <param name="udi">The UDI.</param>
+    /// <param name="checksum">The checksum.</param>
+    /// <param name="dependencies">The artifact dependencies.</param>
     public ArtifactSignature(Udi udi, string checksum, IEnumerable<ArtifactDependency>? dependencies = null)
     {
         Udi = udi;
         Checksum = checksum;
-        Dependencies = dependencies ?? Enumerable.Empty<ArtifactDependency>();
+        Dependencies = dependencies ?? Array.Empty<ArtifactDependency>();
```

Comment:
Why this change?

Reply:
Because `Enumerable.Empty<>()` is just a wrapper over (forwarding call to) `Array.Empty<>()`, see: https://github.com/dotnet/runtime/blob/b95bccb1e6117acb7d64954117672493e35fe6af/src/libraries/System.Linq/src/System/Linq/Enumerable.SizeOpt.cs#L10